config/rbac/role_spi.yaml

```diff
@@ -0,0 +1,24 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: manager-role-spi
+rules:
+- verbs:
+    - '*'
+  apiGroups:
+    - appstudio.redhat.com
+  resources:
+    - spiaccesstokenbindings
+    - spiaccesstokens
+- verbs:
+    - create # allows addition of credentials only.
+    - delete
+    - list
+    - get
+    - watch
```

Comment:
do we need this? I dont see it being used 🤔

Reply:
Needed here https://github.com/redhat-appstudio/application-service/pull/63/files#diff-3fd064f2f1027a954b035a6d5885b96cc5b483f04889d14fa840ace5e9c29c55R33

Otherwise calls to scm-file-retriever will fail with rbac errors